dbatools.psm1

```diff
@@ -125,3 +125,5 @@ Set-Alias -Name Test-SqlDiskAllocation -Value Test-DbaDiskAllocation
 Set-Alias -Name Get-DiskSpace -Value Get-DbaDiskSpace
 Set-Alias -Name Get-SqlMaxMemory -Value Get-DbaMaxMemory
 Set-Alias -Name Set-SqlMaxMemory -Value Set-DbaMaxMemory
+
+. $PSScriptRoot\utils.ps1

```

Comment:
This line should be removed. Once your functions are added to the appropriate folder there is other code that will handle importing them so you can call them.